models/encoding.go

```diff
@@ -378,3 +379,7 @@ type LiveStatusResponse struct {
 	Status    bitmovintypes.ResponseStatus `json:"status,omitempty"`
 	Data      LiveStatusData               `json:"data,omitempty"`
 }
+
+type StartRequest struct {
```

Comment:
Shouldn't this be called `StartOptions`? StartRequest would mean it's the whole request which is not correct here..